app/packages/addons-service/provider.ts

```diff
@@ -197,7 +203,7 @@ export default class Provider {
     }
 
     async userAddonAccounts() {
-        return await this.userReference.authorizedStorageAccounts;
+        return await this.userReference?.authorizedStorageAccounts;
```

Comment:
Is there a way we wouldn't have a userReference?

Reply:
Ahh, good point. I guess in the very off-chance that the request to get the userReference from GV fails, maybe? I don't think this change is needed though, and is a leftover from some earlier type changes that I was trying out?